.github/workflows/docs.yml

```
@@ -34,7 +34,7 @@ jobs:
                 --allow-remove-essential \
                 --allow-change-held-packages \
                 doxygen python3 python3-pip
-            pip install breathe recommonmark sphinx sphinx-material
+            pip install $(cat $PWD/docs/requirements.txt)
```

Is this more idiomatic:
```suggestion
            pip install --requirement $PWD/docs/requirements.txt
```
?